replicate.Rmd

```diff
@@ -0,0 +1,1684 @@
+  round(digits = 2)
+
+clean_2c_low_1000 <- clean_2c %>%
+  filter(split == 0, webercond == 2) %>%
+  summarise(onethousand.go = (sum(go) / n()), CI = (1.96 * Sem(go))) %>%
+  round(digits = 2)
+#producing data.frames for new figure (2c) - low income
+```
+
+
+```{r echo = FALSE}
+title = factor(c("Lower Income", "Higher Income"), levels = c("Lower Income", "Higher Income"))
+values = c(clean_2c_low_300$threehundred.go, clean_2c_high_300$threehundred.go, clean_2c_low_500$fivehundred.go, clean_2c_high_500$fivehundred.go, clean_2c_low_1000$onethousand.go, clean_2c_high_1000$onethousand.go)
+tablet_cost = c("$300", "$300", "$500", "$500", "$1,000", "$1,000")
+CIs = c(clean_2c_low_300$CI, clean_2c_high_300$CI, clean_2c_low_500$CI, clean_2c_high_500$CI, clean_2c_low_1000$CI, clean_2c_high_1000$CI)
+
+fig2c.table <- data.frame(title, values, tablet_cost, CIs)
+#figure 2b data frame
+fig2c.table$tablet_cost <- factor(fig2c.table$tablet_cost, levels = c("$300", "$500", "$1,000"))
+
+figure2c <- fig2c.table %>%
+  ggplot(aes(x = title, y = values, fill = tablet_cost)) + geom_bar(stat = "identity", position = position_dodge(), colour = "black") + ylab("proportion willing to travel") + xlab(NULL) + geom_errorbar(aes(ymin = values - CIs, ymax = values + CIs), width = 0.2, position = position_dodge(0.9)) + ggtitle("Figure 2c - Proportion of participants willing to travel\nfor a discounted tablet computer in Study 2c\n(error bars represent 95% confidence interval)") + theme_bw() + theme(panel.border = element_blank(), panel.grid.major = element_blank(), panel.grid.minor = element_blank(), axis.line = element_line(colour = "black"))
+#produces figure 2c
+
+print(figure2c)
+#displays figure 2c
+```
+
+> When income was treated continuously, the interaction between income and context was clear, beta = -0.39, Wald-test chi square(1, N = 1,893) = 6.13, p < 0.05.
+
+```{r echo = FALSE}
+glm(go ~ logses * webercond, data = clean_2c, family = "binomial") %>%
+  tidy() %>%
+  filter(term == "logses:webercond") %>%
+  mutate(beta = round(estimate, digit = 2),
+         p = round(p.value, digit = 4)) %>%
+  select(term, beta, p)
+
+logit7logses <- glm(go ~ logses * webercond, data = clean_2c, family = "binomial")
+
+print(wald.test(b = coef(logit7logses), Sigma = vcov(logit7logses), Terms = 4)$result$chi2[1], digit = 3)
+```
+
+---
+
+### Study 2d 
+
+> In Study 2d, there were 301 participants (mean age = 30.2 years; 120 females, 181 males; median household size = 3 people; median household income = $25,000).
+
+```{r echo = FALSE}
+# Read data in
+suppressMessages(suppressWarnings(df2d_1 <- read.csv("csv_files/2d_Sheet1.csv", header=TRUE)))
+suppressMessages(suppressWarnings(df2d_2 <- read.csv("csv_files/2d_Sheet2.csv", header=TRUE)))
+suppressMessages(suppressWarnings(df2d_spss <- read.spss("framing/study_2d.sav", to.data.frame = TRUE)))
+
+```
+
+```{r echo = FALSE}
+# Number of subjects
+df2d_1 %>%
+  select(subject) %>%
+  summarise(n())
+
```

Comment:
sheet 1 and sheet 2 from an xlsx file we got - will add a comment where these data.frames are assigned